P6-Gameplay-Part-2/content.md

```diff
@@ -76,10 +76,8 @@ A great place to add a steady decline would be in the `update(..)` method.
 override func update(currentTime: CFTimeInterval) {
   /* Called before each frame is rendered */
   if state != .Playing { return }
-
```

Comment:
I understand, I removed the line because Markdown was breaking the code block, that is everything after the lone break ended up outside the code block and looked like a normal paragraph.